### ISSUE TITLE
SeqTilePatternTest: Missing parameter in LOG_MESSAGE

### ISSUE THREAD

**Issue by devreal** [dash-project/dash#162]
This line (SeqTilePatternTest.cc:29) causes invalid reads:
```
  LOG_MESSAGE("e:%d,%d, bs:%d,%d, nu:%d, mpu:%d",
              size_rows,  size_cols,
              block_rows, block_cols,
              team_size);
```

Just count the number of expected and provided parameters ;) I just can't interpret the cryptic output so I cannot provide a fix...

**Comment by fuchsto** [comment]
Aah. Just delete that log message then, it's too verbosive for non-debugging purposes anyways.

Oh, and *ob-vi-ous-ly* it's

e: extents
bs: block size
nu: number of units
mpu: ... eh ... something per unit, beats me.

**Comment by fmoessbauer** [comment]
Merged in #164. Ticket can be closed
